polymer.externs.js

```diff
@@ -443,6 +443,12 @@ PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 PolymerDomApi.prototype.classList;
 
 /**
+ * @param {string} selector
+ * @return {!Array<?HTMLElement>}
```

Comment:
Hmm I actually think that should probably be non-null as well... Let's just make both non-null.